server/routes/viewingLicences/handlers/printLicence.test.ts

```diff
@@ -26,7 +26,7 @@ describe('Route - print a licence', () => {
     qrCodeService.getQrCode = jest.fn()
     licenceService.getExclusionZoneImageData = jest.fn()
     licenceService.recordAuditEvent = jest.fn()
-    hdcService.getHdcInfo = jest.fn()
+    hdcService.getHdcLicenceData = jest.fn()
```

Comment:
**suggestions:(non-blocking)** we should probably turn these into proper jest mocks, I've been tidying them up as I hit them